exporter/device.go

```diff
@@ -0,0 +1,38 @@
+package exporter
+
+import (
+	"context"
+	"time"
+
+	"github.com/ffddorf/unms-exporter/client/devices"
+	"github.com/ffddorf/unms-exporter/models"
+)
+
+type Device struct {
+	*models.DeviceStatusOverview
+}
+
+func (e *Exporter) fetchDeviceData() ([]Device, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	params := &devices.GetDevicesParams{
+		WithInterfaces: &defaultWithInterfaces,
```

Comment:
did you mean to use `e.withInterfaces` here?

Reply:
Oh, yeah. In #23 this is already fixes, I guess I missed this while rebasing. I'll push an update in a minute.